### DISCUSSION TITLE
Styling of cursor _additions_

### DISCUSSION THREAD

**Discussion by Benni3-1415** [ubuntu/yaru#2929] [General]
**additions:** the small extra icons at the bottom of some cursors who are indicating a special action ![addition](https://user-images.githubusercontent.com/73243667/122967368-ae690d80-d38a-11eb-8890-810f30ef73f8.png)
**dark:** black / orange

#### Are they cursors?
IMO no, because the shadows would visually overload the cursors.
The `+` sign would have a low contrast on orange and the spinner _addition_ would need to be inverted too.

#### White on dark or dark on white? (if they aren't cursors)
I think white on dark is the best, to clearly show, that this is an _addition_ and not a cursor.
If we chose dark on white users could wonder: _Why are there no shadows?_

White on dark was also used in master (in most _additions_) until @Jupi007 changed it in #2926.

### Suggestion (white on dark)
![source-cursors](https://user-images.githubusercontent.com/73243667/122966984-44e8ff00-d38a-11eb-8c31-e462fc24ad30.png)

I would create a PR for it.

**Comment by Jupi007** [comment]
🤔 Indeed _additions_ are not cursors, but they are part of it.
Personally I prefer current dark on white _additions_ style because it looks sharper on light background (which is the main one). That's why I unified this look on my last PR.
But we can quite easily considering using white on dark instead, just the spinner _addition_ looks a bit bad with solid dark background, I really prefer gradient.

Just as reminder, here are current cursors:

![Capture d’écran du 2021-06-25 16-00-14](https://user-images.githubusercontent.com/36476595/123435976-718d5880-d5ce-11eb-8ba6-e71623723aea.png)

**Reply by Benni3-1415** [reply]
> ... just the spinner addition looks a bit bad with solid dark background, I really prefer gradient.

In the current variant (dark on white) the _pointer watch_ cursor would look like this:
![source-cursors](https://user-images.githubusercontent.com/73243667/123439149-b961af00-d5d1-11eb-8796-c3d70cdef290.png)

The _copy_ cursor:
![source-cursors2](https://user-images.githubusercontent.com/73243667/123440743-6688f700-d5d3-11eb-8069-4e47ea56242d.png)

**Reply by Jupi007** [reply]
IMO not, these two cursors are special case.
The spinner should use the same background gradient as the spinner cursor to keep consistent.
And the copy orange circle looks better with the white plus.

Honestly I think current cursors colors looks perfect :)

**Comment by swarajkothekar** [comment]
than index finger is little long compared to the one in ubuntu 21.04

**Reply by Benni3-1415** [reply]
Hello @kothekarswaraj,
did you mean the `hand2` / `pointer` cursor?

### The new cursor with the short index finger:
![hand2-index-large](https://user-images.githubusercontent.com/73243667/127372333-7e483d0a-b71e-4c2b-a8bc-090806cad85c.png)
![hand2-index](https://user-images.githubusercontent.com/73243667/127372344-38680e8a-3251-49d8-a82a-164ea5087b8b.png)

IMO, the short index finger is too small compared to the hand (1: 2) because the optical size of the hand has increased in the new version …
